devel/grails/Portfile

```diff
@@ -59,3 +59,5 @@ notes "
 Remember to set the environment variable GRAILS_HOME to the path to\
 the grails distribution: ${prefix}/share/java/${name}
 "
+
+livecheck.regex     archive/refs/tags/v(\[^\"M\]+)\.tar\.gz
```

Comment:
can you not set the "usual" `github.livecheck.regex  {([0-9.-]+)}` for this purpose?

Reply:
It's work, thanks 🌟